vpns.html

```diff
@@ -32,7 +32,7 @@ <h1>VPNs</h1>
     If you want privacy and anonymity, use <a href="https://www.torproject.org/">Tor</a> instead, which avoids trusting a single party and is more private by design. <br>
     <br>
     Also see <a href="https://gist.github.com/joepie91/5a9909939e6ce7d09e29">"Don't use VPN services"</a> and <a
-    href="https://matt.traudt.xyz/posts/you-want-tor-24tFBCJV.html">"You want Tor Browser... not a VPN"</a>.
+    href="https://matt.traudt.xyz/posts/2019-10-17-you-want-tor-browser-not-a-vpn/"</a>.
```

Comment:
This is not valid. It's missing syntax and you need to fill in the tag.

Reply:
Yep sorry, big fail